_posts/2021-02-01-launching-the-solidity-forum.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+published: true
+title: 'Launching the Solidity Forum 🗃️'
+date: '2021-02-01'
+author: Franziska Heintel
+category: Announcements
+---
+
+In our effort to foster exchange of information, encourage more developers to give feedback about Solidity and join the discussions on language design and future direction of the compiler, we are happy to launch the [Solidity forum](https://forum.soliditylang.org/) today! 
+
+Moving forward the Solidity forum will be the dedicated place to discuss topics and questions related to…
+
+- The implementation and design of Solidity programming language.
+- The Solidity compiler.
+- Useful Solidity tips and code snippets.
+- Solidity documentation and its translation.
+- Discussions and announcements about Solidity releases.
+
+It will **not** be the place for…
+
+- Ad-hoc support questions. For urgent Solidity support questions, please use the [Solidity Gitter chat](https://gitter.im/ethereum/solidity).
+- Solidity bugs, vulnerabilities or issues. To report a bug, please use the [GitHub issues tracker](https://github.com/ethereum/solidity/issues) and refer to this guide on [how to report issues](https://docs.soliditylang.org/en/latest/contributing.html#how-to-report-issues).
+- Generic Ethereum discussion. For that visit [r/ethereum](https://www.reddit.com/r/ethereum).
+- Ethereum research specific discussion. For that, please refer to the [ethresear.ch](https://ethresear.ch/) forum.
+- EIP specific discussion. For that visit the [Ethereum Magicians](https://ethereum-magicians.org/) forum.
+
+The Solidity forum is based on [Discourse](https://www.discourse.org/about), an open-source discussion platform for civilized discussions. :)
+
+If you are new to Discourse consider having a look at this [“Unofficial Discourse User Reference Guide”](https://forum.digikey.com/t/an-unofficial-discourse-user-reference-guide/1125), which summarizes useful tips on how to use Discord, text formatting, images, LaTex and more.
+ 
+### A Quick Guide to the Solidity Forum Categories 🔖
+
+We are launching the forum with an initial selection of categories which can be adjusted and extended over time. To propose a new category, or the change of an existing one, please post your suggestions in the [“Site Feedback”](https://forum.soliditylang.org/c/site-feedback/2) category. 
+
+#### → Announcements
+[”Announcements”](https://forum.soliditylang.org/c/announcements/5) is a low-traffic category for important announcements about the Solidity language and compiler.
+
+📮 Subscribe to this category if you want to be kept in the loop about releases, security updates and Solidity-relevant feedback surveys, news and events.
+
+#### → Solidity Compiler
+[“Solidity Compiler”](https://forum.soliditylang.org/c/solidity-compiler/6) is a category for discussions and feedback about the Solidity compiler.
+
```

Comment:
Maybe we should also give examples for each of the categories? It's not fully clear to me what issues about the compiler could be. This could be misleading to mean "implementation of the language".

I assume this is about the different input and output data structures the compiler generates - is that true? Should it then rather be "tool support" or something?

Reply:
Hmm "tool support" I would rather see in tooling and infrastructure? Or maybe not? I mean those topics as is were only a proposal from my side, happy to adjust them..

Reply:
Sure, but what did you have in mind for this category?

Reply:
I would also prefer more categories than fewer, but I can't think of a good division right now.

Reply:
I'm not entirely sure, to be perfectly honest I think I copied this category from the Rust blog. They defined it as the following: "Discussion about the compiler itself – internal data structures and algorithms, optimizations, parsing, and so on. Home of the compiler subteam."

Reply:
Ah right, but I think we said we won't discuss such things in the forum, right?

Reply:
Ok so remove this category?